managed/ui/src/components/universes/UniverseDetail/UniverseDetail.js

```diff
@@ -582,6 +595,45 @@ class UniverseDetail extends Component {
                           Encryption at-Rest
                         </YBMenuItem>
                       </>
+                    ),
+                    universeControls: (backToMainMenu) => (
+                      <>
+                        <MenuItem onClick={backToMainMenu}>
+                          <YBLabelWithIcon icon="fa fa-chevron-left fa-fw">Back</YBLabelWithIcon>
+                        </MenuItem>
+                        <MenuItem divider />
+
+                        {!universePaused &&
+                          <YBMenuItem
+                            onClick={showRollingRestartModal}
+                            availability={getFeatureState(
+                              currentCustomer.data.features,
+                              'universes.details.overview.restartUniverse'
+                            )}
+                          >
+                            Initiate Rolling Restart
+                          </YBMenuItem>
+                        }
+
+                        {/* TODO: 1 more condition needs to be added which specifies the
+                        current status of the universe. */}
+                        {isDefinedNotNull(providerType)
+                          && providerType.toString() === 'aws' &&
```

Comment:
- `providerType` should be string by definition
- I'd recommend having smth like `isUniversePausable()` and put all logic inside that function

Reply:
I've updated the condition.